src/mscorlib/src/Interop/Windows/Kernel32/Interop.GetSystemDirectoryW.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, SetLastError = true)]
+        private static extern unsafe int GetSystemDirectoryW(char* lpBuffer, int jSize);
```

Comment:
Where is the `j` in `jSize` from?  In the native method it's named `uSize`.

Reply:
I noticed that too, thought about changing it to match the native method, but just left it as it was in CoreFX. This is an exact copy of the file from CoreFX.